Dockerfile.prod

```diff
@@ -33,7 +33,16 @@ RUN bundle install --without development test
 
 COPY . /usr/src/app/
 
+# For ActiveAdmin assets
 # https://github.com/rails/rails/issues/32947#issuecomment-511532631
 RUN SECRET_KEY_BASE=dummy bin/rails assets:precompile
 
+# For puma server
 RUN mkdir -p tmp/pids
+
+# Build vue files
+COPY vue/package*.json vue/yarn.lock /usr/src/app/vue/
```

Comment:
We shouldn't allow both package-lock.json and yarn.lock

Reply:
@jhash which one are you using?